invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Home/components/lists/OverdueLoansList/state/actions.js

```diff
@@ -12,6 +12,7 @@ export const fetchOverdueLoans = () => {
         loanApi
           .query()
           .overdue()
+          .withState('ITEM_ON_LOAN')
```

Comment:
We managed to avoid hardcoding `ITEM_ON_LOAN` until now... but I don't have a better solution...

Reply:
we could read all of the states from config, like in the other cases

Reply:
I think is better to address that after when we will have a more clear picture of what is needed to be in the config. Except if is clear that goes there :)